src/client/components/equalizer/equalizer.js

```diff
@@ -232,19 +233,19 @@ class Equalizer extends Component {
       <Button
         className="ButtonStyleTemplate"
         onClick={playSoundFromFile}
-        icon={this.props.audioData.loading ? null : playIcon}
-        value={this.props.audioData.loading
-          ? <img src={Spinner} alt="Play music spinner" /> : 'Play'}
-        disabled={this.props.audioData.loading ? 'disabled' : null}
+        icon={loading ? null : playIcon}
+        value={loading ? <img src={Spinner} alt="Play music spinner" /> : 'Play'}
+        disabled={loading ? 'disabled' : null}
       />
     );
 
     const PauseButton = (
       <Button
         className="ButtonStyleTemplate"
         onClick={pauseSoundFromFile}
-        icon={pauseIcon}
-        value="Pause"
+        icon={loading ? null : pauseIcon}
+        value={loading ? <img src={Spinner} alt="Play music spinner" /> : 'Pause'}
+        disabled={loading ? 'disabled' : null}
```

Comment:
Here also you can add loading && ‘disabled’

Reply:
Done)